modules/reindex/src/main/java/org/elasticsearch/reindex/Reindexer.java

```diff
@@ -82,26 +83,31 @@ public class Reindexer {
     private final ThreadPool threadPool;
     private final ScriptService scriptService;
     private final ReindexSslConfig reindexSslConfig;
+    private final ReindexMetrics reindexMetrics;
 
     Reindexer(
         ClusterService clusterService,
         Client client,
         ThreadPool threadPool,
         ScriptService scriptService,
-        ReindexSslConfig reindexSslConfig
+        ReindexSslConfig reindexSslConfig,
+        ReindexMetrics reindexMetrics
```

Comment:
I think you missed adding this class when commiting the changes.

Reply:
Ah Yes! Sorry about that.